ballerina-tests/http-service-tests/tests/anydata_union_return_type_service.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+import http_service_tests.records;
```

Comment:
Any specific reason to add the records to a separate module?

Reply:
Added that to simulate the original use case

Reply:
Ah ok, in that case can we also run the `GraalVM Check` on your branch to see whether this is working without any issue?

Reply:
Passing https://github.com/dilanSachi/module-ballerina-http/actions/runs/4013386586